test/test_crowdsale.js

```diff
@@ -0,0 +1,121 @@
+const MFC_Token = artifacts.require("MFC_Token");
+const WhitelistedCrowdsale = artifacts.require("WhitelistedCrowdsale");
+
+contract('ICO_crowdsale test', async (accounts) => {
+    const owner = accounts[0];
+    const user = accounts[1];
+
+    const ether_0 = web3.toWei(0, 'ether');
+    const ether_5 = web3.toWei(5, 'ether');
+    const ether_10 = web3.toWei(10, 'ether');
+    const ether_25 = web3.toWei(25, 'ether');
+    const ether_100 = web3.toWei(100, 'ether');
+
+    const rate = 10;
+
+    let weiAmount = ether_0;
+
+    it('should add nothing to bonus amount for with 0 <= weiAmount < 10 ', async function () {
+        weiAmount = ether_5;
+
+        let basicAmount = weiAmount * rate;
+        let bonusAmount = 0;
+        let result = basicAmount + bonusAmount;
+
+        let contract = await WhitelistedCrowdsale.deployed();
+
+        let balance = await web3.eth.getBalance(owner);
+        let balance_before = web3.fromWei(balance.toNumber(), 'ether');
+
+        await web3.eth.sendTransaction({from: user, to: owner, value: weiAmount, gasLimit: 6721975, gasPrice: 20000000000});

```

Comment:
Why do you make these transactions?
Don't you need just to check right token calculation

Reply:
I wanted to create a test for the transaction amount because weiAmount is value of transaction. Well this is also understandable. I will fix it.